test/utils/SQLHelpers.ts

```diff
@@ -39,7 +38,19 @@ describe("SQLHelpers", function () {
         "2, 'test', 'information'"
       )
     ).to.equal(
-      "INSERT INTO test_101_31337_1 (id, name, desc) VALUES (2, 'test', 'information')"
+      "INSERT INTO test_101_31337_1(id, name, desc)VALUES(2, 'test', 'information')"
+    );
+  });
+
+  it("Should return a valid INSERT statement from columns and an array of values", async function () {
+    await expect(
+      // This is not a valid name in Tableland but tests string concat.
+      await lib.toBatchInsert("test_101", 1, "id, name, desc", [
+        "1,'hello','information'",
+        "2,'world','information'",
+      ])
+    ).to.equal(
+      "INSERT INTO test_101_31337_1(id, name, desc)VALUES(1,'hello','information'),(2,'world','information')"
```

Comment:
possible to remove the spaces between the column names?

Reply:
totally, good catch. will update

Reply:
all done